README.md

```diff
@@ -1,14 +1,26 @@
-# Mesh Angular Demo
+# Gentics Mesh Angular Demo
 
-This is a simple example application intended to demonstrate a few of the features of Gentics Mesh 
-([getmesh.io](http://getmesh.io)).
+<img src="/gentics-mesh-angular-example-screenshot.png" align="center" width="100%"><br/>
+This is a simple example application intended to demonstrate a few of the features of [Gentics Mesh](https://getmesh.io/)
+.
 
-All calls to the Mesh API endpoints are made from the [`MeshService`](app/mesh.service.ts). The rest of the 
-app logic is contained in the [app component](app/app.component.ts) file.
+## How to run the example
+The example is implemented with Angular 5.x and built with the [Angular CLI](https://github.com/angular/angular-cli).
 
-The app is using Angular and built with [Angular CLI](https://github.com/angular/angular-cli). Run `ng serve`
-to start a dev server at `http://localhost:4200/`, `ng build --dist` to build the project to `dist/`.
+You will need a locally running instance of the [Gentics Mesh](https://getmesh.io/) demo. You can use docker for that:
+``` 
+docker run -p 8080:8080 gentics/mesh-demo
+``` 
+Check the [documentation](https://getmesh.io/docs/beta/administration-guide.html#_run_with_docker) for details and other options.
 
-This is intended to be an extremely simple example. Therefore the implementations
-are necessarily naive. The idea is to provide a way to see the Mesh API in action and provide an example for
+Once the Gentics Mesh local instance is up and running you can start the example with
+`npm run-script start` which will start a dev server at [http://localhost:4200/](http://localhost:4200/).
```

Comment:
the URL would be `http://localhost:4200/demo` (but I'll check back, as [per this comment](https://github.com/gentics/mesh-angular-example/pull/6#issuecomment-377480043))

Reply:
Done in 9552f8e